Fast search option: search in SearchableText

mauritsvanrees opened the pull request:
This branch is based on pull request #28: it includes those commits. They are not really related, but they partly touch the same code.

This adds an option in the search form. I have [another branch](https://github.com/collective/collective.searchandreplace/tree/maurits-searchabletext-in-controlpanel) where I initially added this option in the control panel instead. In the form it is more flexible as the end user has the choice. This has the automatic downside that the end user has yet another option, making the form more complex. But at least the search/replacing is now faster by default. And a Manager will not get requests to temporarily switch the setting in the control panel.

@Gagaro Do you agree with this change, and with putting it in the form?

Gagaro commented:
I would add in the description that only keyword will be searched, and that HTML tag won't be (HTML replacing is the main use of this module by my client).

mauritsvanrees commented:
Ah, good point about replacing html tags. I have updated the description and also added a test for it. Plus some stuff I overlooked in this pull request. I will update the readme too, and then I intend to merge and release 7.0.
